src/test/regress/sql/create_drop_database_propagation.sql

```diff
@@ -0,0 +1,121 @@
+
+
+\set create_drop_db_tablespace :abs_srcdir '/tmp_check/ts3'
+CREATE TABLESPACE create_drop_db_tablespace LOCATION :'create_drop_db_tablespace';
+
+\c - - - :worker_1_port
+\set create_drop_db_tablespace :abs_srcdir '/tmp_check/ts4'
+CREATE TABLESPACE create_drop_db_tablespace LOCATION :'create_drop_db_tablespace';
+
+\c - - - :worker_2_port
+\set create_drop_db_tablespace :abs_srcdir '/tmp_check/ts5'
+CREATE TABLESPACE create_drop_db_tablespace LOCATION :'create_drop_db_tablespace';
+
+\c - - - :master_port
+create user create_drop_db_test_user;
+
+set citus.enable_create_database_propagation=on;
+
+CREATE DATABASE mydatabase
+    WITH TEMPLATE = 'template0'
+            OWNER = create_drop_db_test_user
+            CONNECTION LIMIT = 10
+            ENCODING = 'UTF8'
```

Comment:
can we use another config for encoding other than `UTF8`, maybe `UTF32`, to test a bit unusual setting?

Reply:
Finding a valid encoding name is difficult since it is strictly depdend on OS capabilities in encoding.